rules/otp.bzl

```diff
@@ -51,6 +51,7 @@ def get_otp_images():
         "//hw/ip/otp_ctrl/data:img_rma",
         "//hw/ip/otp_ctrl/data:img_test_unlocked0",
         "//hw/ip/otp_ctrl/data:img_prod",
+        "//hw/ip/otp_ctrl/data:img_prod_end",
```

Comment:
This change was merged a few hours ago as part of #16034.

Reply:
Thanks for pointing this out. Yes, I noticed that while I was rebasing since git dropped the NOP commit by itself.